src/index.js

```diff
@@ -240,16 +240,20 @@ rtm.on(RTM_EVENTS.MESSAGE, (message) => {
     }
   } else if (message.text.toLowerCase() === 'help') {
     rtm.sendMessage('*command list*\n' +
-        '> feel [emoji] [1-6] ([snippet]) `tell the bot how you feel now, snippet is optional`\n' +
+        '> feel/felt [emoji] [1-6] ([snippet]) `tell the bot how you feel/felt now, snippet is optional`\n' +
```

Comment:
> ...how you feel/felt now...
This is a bit confusing. 
What about showing them as separate commands in the help?

Reply:
yes! good suggestion.